baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
What happens when we fire up multiple instances of Maestro? Will there be collisions?